lib/routes/home/widgets/payments_list/widgets/payment_details_sheet/widgets/payment_details_sheet_refund_tx_fee_amount.dart

```diff
@@ -53,7 +54,7 @@ class PaymentDetailsSheetRefundTxAmount extends StatelessWidget {
                   state.bitcoinTicker,
                 ).format(refundTxAmountSat);
                 return Text(
-                  paymentData.paymentType == PaymentType.receive
+                  paymentData.paymentType == PaymentType.receive || paymentData.isRefunded
```

Comment:
I feel like it makes more sense to keep the amount negative and the refund positive. As the amount left the wallet and refund returned to the wallet